README.md

```diff
@@ -53,19 +53,24 @@ Clone the docs repo into a directory you want the ODK-X docs files to be located
 1. Visit this link https://github.com/odk-x/docs on your browser to locate the docs repo
 2. Find and click the "fork" button to create a personal fork of the project on Github
 3. Navigate to the directory you want the files to be located using the "cd" (Change Directory) command on a command-line interface on your local machine:
+
 ```
 cd <DIRECTORY>
 ```
-4. Get a copy of the ODK-X docs repository on your local machine by cloning the forked repo to your local machine from your github account using the clone command below. Replace "LINK-TO-YOUR-FORKED-REPO" below with the actual link to your forked repo:  
- 
+
+4. Get a copy of the ODK-X docs repository on your local machine by cloning the forked repo to your local machine from your github account using the clone command below. Replace "LINK-TO-YOUR-FORKED-REPO" below with the actual link to your forked repo: 
+
+Alternatively:
+Copy the forked ODK-X docs repository URL by clicking on the green "Code" button on Github and navigate to the command-line interface on your local machine.
```

Comment:
@Chioma-data-analytics You need to click on the green "Code" button to get the "LINK-TO-YOUR-FORKED-REPO"

When making documentation, it's best to try it out yourself BEFORE writing about it.

Reply:
Hi @wbrunette apologies for the break and thank you for guiding me through. 

Just to confirm my understanding, you mean I use "You need to click on the green "Code" button to get the "LINK-TO-YOUR-FORKED-REPO" instead of  "Copy the forked ODK-X docs repository URL by clicking on the green "Code" button on Github and navigate to the command-line interface on your local machine" is my understanding correct?

Reply:
I am confused by your question. Please put your proposal into the PR and we will review. Also please try these steps out and make sure it makes sense to you.